mdbook: 0.4.52 -> 0.5.1

roberth opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## TODO

- [x] https://github.com/NixOS/nix/pull/14690
- [x] backport or patch Nixes
- [x] check that Lix fix is also available in Nixpkgs
- [x] devenv
- [x] `engage` 
- [x] `helix` 
- [x] `evil-helix` 
- [x] `mdbook-cmdrun`  https://github.com/FauconFan/mdbook-cmdrun/pull/23

Alternatively, `mdbook_0_4` could be added.

## Things done


<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

## Context

- Successor of https://github.com/NixOS/nixpkgs/pull/462777#issuecomment-3581149290

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

roberth committed:
mdbook: 0.4.52 -> 0.5.1

roberth committed:
nixVersions: apply mdbook 0.5 support patches

roberth committed:
lixPackageSets: apply mdbook 0.5 support patches

Apply patches to lix_2_93, lix_2_94, and git versions to support
building documentation with mdbook 0.5.

The patch is from upstream Lix commit 54df89f6, which maintains
compatibility with both mdbook 0.4.x and 0.5.x.

https://git.lix.systems/lix-project/lix/issues/1051

philiptaron reviewed (dismissed):
It's kind of messy, but I think this works. Doing a build.

roberth commented:
> `staging-nixos`

Done.

> Do we need to update the fallback Nix paths?

The process for that is to take them from releases.nixos.org, where they haven't changed, so it would be a no-op.
I think it was acceptable for those to be a bit out of sync, but maybe we should
- https://github.com/NixOS/nixpkgs/issues/471439?

philiptaron commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 467009`
Commit: `5cd20ff81da5232849eb782eb47af4078f606273`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 1 package marked as broken and skipped:</summary>
  <ul>
    <li>tests.pkg-config.defaultPkgConfigPackages.tests-combined</li>
  </ul>
</details>
<details>
  <summary>:fast_forward: 2 packages blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>:x: 16 packages failed to build:</summary>
  <ul>
    <li>devenv</li>
    <li>engage</li>
    <li>engage.doc</li>
    <li>evil-helix</li>
    <li>helix</li>
    <li>helix.doc</li>
    <li>mdbook-cmdrun</li>
    <li>nixStatic</li>
    <li>nixStatic.dev</li>
    <li>nixStatic.doc</li>
    <li>nixStatic.man</li>
    <li>nixVersions.git</li>
    <li>nixVersions.git.dev</li>
    <li>nixVersions.git.doc</li>
    <li>nixVersions.git.man</li>
    <li>tests.devShellTools.nixos</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 test built:</summary>
  <ul>
    <li>nixosTests.simple</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 373 packages built:</summary>
  <ul>
    <li>aba</li>
    <li>appvm</li>
    <li>attic-client</li>
    <li>bundix</li>
    <li>cabal2nix</li>
    <li>cached-nix-shell</li>
    <li>cachix (cachix.bin)</li>
    <li>catppuccin-cursors</li>
    <li>catppuccin-cursors.frappeBlue</li>
    <li>catppuccin-cursors.frappeDark</li>
    <li>catppuccin-cursors.frappeFlamingo</li>
    <li>catppuccin-cursors.frappeGreen</li>
    <li>catppuccin-cursors.frappeLavender</li>
    <li>catppuccin-cursors.frappeLight</li>
    <li>catppuccin-cursors.frappeMaroon</li>
    <li>catppuccin-cursors.frappeMauve</li>
    <li>catppuccin-cursors.frappePeach</li>
    <li>catppuccin-cursors.frappePink</li>
    <li>catppuccin-cursors.frappeRed</li>
    <li>catppuccin-cursors.frappeRosewater</li>
    <li>catppuccin-cursors.frappeSapphire</li>
    <li>catppuccin-cursors.frappeSky</li>
    <li>catppuccin-cursors.frappeTeal</li>
    <li>catppuccin-cursors.frappeYellow</li>
    <li>catppuccin-cursors.latteBlue</li>
    <li>catppuccin-cursors.latteDark</li>
    <li>catppuccin-cursors.latteFlamingo</li>
    <li>catppuccin-cursors.latteGreen</li>
    <li>catppuccin-cursors.latteLavender</li>
    <li>catppuccin-cursors.latteLight</li>
    <li>catppuccin-cursors.latteMaroon</li>
    <li>catppuccin-cursors.latteMauve</li>
    <li>catppuccin-cursors.lattePeach</li>
    <li>catppuccin-cursors.lattePink</li>
    <li>catppuccin-cursors.latteRed</li>
    <li>catppuccin-cursors.latteRosewater</li>
    <li>catppuccin-cursors.latteSapphire</li>
    <li>catppuccin-cursors.latteSky</li>
    <li>catppuccin-cursors.latteTeal</li>
    <li>catppuccin-cursors.latteYellow</li>
    <li>catppuccin-cursors.macchiatoBlue</li>
    <li>catppuccin-cursors.macchiatoDark</li>
    <li>catppuccin-cursors.macchiatoFlamingo</li>
    <li>catppuccin-cursors.macchiatoGreen</li>
    <li>catppuccin-cursors.macchiatoLavender</li>
    <li>catppuccin-cursors.macchiatoLight</li>
    <li>catppuccin-cursors.macchiatoMaroon</li>
    <li>catppuccin-cursors.macchiatoMauve</li>
    <li>catppuccin-cursors.macchiatoPeach</li>
    <li>catppuccin-cursors.macchiatoPink</li>
    <li>catppuccin-cursors.macchiatoRed</li>
    <li>catppuccin-cursors.macchiatoRosewater</li>
    <li>catppuccin-cursors.macchiatoSapphire</li>
    <li>catppuccin-cursors.macchiatoSky</li>
    <li>catppuccin-cursors.macchiatoTeal</li>
    <li>catppuccin-cursors.macchiatoYellow</li>
    <li>catppuccin-cursors.mochaBlue</li>
    <li>catppuccin-cursors.mochaDark</li>
    <li>catppuccin-cursors.mochaFlamingo</li>
    <li>catppuccin-cursors.mochaGreen</li>
    <li>catppuccin-cursors.mochaLavender</li>
    <li>catppuccin-cursors.mochaLight</li>
    <li>catppuccin-cursors.mochaMaroon</li>
    <li>catppuccin-cursors.mochaMauve</li>
    <li>catppuccin-cursors.mochaPeach</li>
    <li>catppuccin-cursors.mochaPink</li>
    <li>catppuccin-cursors.mochaRed</li>
    <li>catppuccin-cursors.mochaRosewater</li>
    <li>catppuccin-cursors.mochaSapphire</li>
    <li>catppuccin-cursors.mochaSky</li>
    <li>catppuccin-cursors.mochaTeal</li>
    <li>catppuccin-cursors.mochaYellow</li>
    <li>catppuccin-sddm</li>
    <li>celeste</li>
    <li>chirpstack-concentratord</li>
    <li>colmena</li>
    <li>comma</li>
    <li>common-updater-scripts</li>
    <li>corto</li>
    <li>cosmic-applets</li>
    <li>cosmic-applibrary</li>
    <li>cosmic-bg</li>
    <li>cosmic-comp</li>
    <li>cosmic-comp.debug</li>
    <li>cosmic-design-demo</li>
    <li>cosmic-edit</li>
    <li>cosmic-ext-applet-caffeine</li>
    <li>cosmic-ext-applet-external-monitor-brightness</li>
    <li>cosmic-ext-applet-minimon</li>
    <li>cosmic-ext-applet-privacy-indicator</li>
    <li>cosmic-ext-applet-sysinfo</li>
    <li>cosmic-ext-applet-weather</li>
    <li>cosmic-ext-calculator</li>
    <li>cosmic-ext-tweaks</li>
    <li>cosmic-files</li>
    <li>cosmic-greeter</li>
    <li>cosmic-icons</li>
    <li>cosmic-idle</li>
    <li>cosmic-initial-setup</li>
    <li>cosmic-launcher</li>
    <li>cosmic-notifications</li>
    <li>cosmic-osd</li>
    <li>cosmic-panel</li>
    <li>cosmic-player</li>
    <li>cosmic-randr</li>
    <li>cosmic-reader</li>
    <li>cosmic-screenshot</li>
    <li>cosmic-session</li>
    <li>cosmic-settings</li>
    <li>cosmic-store</li>
    <li>cosmic-term</li>
    <li>cosmic-workspaces-epoch</li>
    <li>cosmic-workspaces-epoch.debug</li>
    <li>crate2nix</li>
    <li>crush</li>
    <li>dhall-nix</li>
    <li>dhall-nixpkgs</li>
    <li>disko</li>
    <li>dogdns</li>
    <li>dogdns.man</li>
    <li>dub-to-nix</li>
    <li>dydisnix</li>
    <li>examine</li>
    <li>find-billy</li>
    <li>forecast</li>
    <li>fusionInventory</li>
    <li>gcalcli</li>
    <li>gcalcli.dist</li>
    <li>gclient2nix</li>
    <li>glpi-agent</li>
    <li>haskellPackages.cachix</li>
    <li>haskellPackages.cachix.bin</li>
    <li>haskellPackages.cachix.doc</li>
    <li>haskellPackages.cli-nix</li>
    <li>haskellPackages.cli-nix.doc</li>
    <li>haskellPackages.dhall-nix</li>
    <li>haskellPackages.dhall-nix.doc</li>
    <li>haskellPackages.dhall-nixpkgs</li>
    <li>haskellPackages.dhall-nixpkgs.doc</li>
    <li>haskellPackages.hercules-ci-agent</li>
    <li>haskellPackages.hercules-ci-agent.doc</li>
    <li>haskellPackages.hercules-ci-cli</li>
    <li>haskellPackages.hercules-ci-cli.doc</li>
    <li>haskellPackages.hercules-ci-cnix-expr</li>
    <li>haskellPackages.hercules-ci-cnix-expr.doc</li>
    <li>haskellPackages.hercules-ci-cnix-store</li>
    <li>haskellPackages.hercules-ci-cnix-store.doc</li>
    <li>haskellPackages.hnix</li>
    <li>haskellPackages.hnix.doc</li>
    <li>haskellPackages.niv</li>
    <li>haskellPackages.niv.bin</li>
    <li>haskellPackages.niv.data</li>
    <li>haskellPackages.niv.doc</li>
    <li>haskellPackages.nix-paths</li>
    <li>haskellPackages.nix-paths.doc</li>
    <li>haskellPackages.nix-serve-ng</li>
    <li>haskellPackages.nix-serve-ng.doc</li>
    <li>haskellPackages.nix-thunk</li>
    <li>haskellPackages.nix-thunk.doc</li>
    <li>haskellPackages.nvfetcher</li>
    <li>haskellPackages.nvfetcher.doc</li>
    <li>haskellPackages.update-nix-fetchgit</li>
    <li>haskellPackages.update-nix-fetchgit.doc</li>
    <li>hci</li>
    <li>hercules-ci-agent</li>
    <li>home-manager</li>
    <li>hydra</li>
    <li>hydra.doc</li>
    <li>just</li>
    <li>just.doc</li>
    <li>just.man</li>
    <li>kabeljau</li>
    <li>kcl</li>
    <li>libcosmicAppHook</li>
    <li>libnixxml</li>
    <li>lix (lixPackageSets.lix_2_93.lix, lixPackageSets.stable.lix)</li>
    <li>lix.debug (lixPackageSets.lix_2_93.lix.debug, lixPackageSets.stable.lix.debug)</li>
    <li>lix.dev (lixPackageSets.lix_2_93.lix.dev, lixPackageSets.stable.lix.dev)</li>
    <li>lix.devdoc (lixPackageSets.lix_2_93.lix.devdoc, lixPackageSets.stable.lix.devdoc)</li>
    <li>lix.doc (lixPackageSets.lix_2_93.lix.doc, lixPackageSets.stable.lix.doc)</li>
    <li>lix.man (lixPackageSets.lix_2_93.lix.man, lixPackageSets.stable.lix.man)</li>
    <li>lixPackageSets.git.colmena</li>
    <li>lixPackageSets.git.lix</li>
    <li>lixPackageSets.git.lix.debug</li>
    <li>lixPackageSets.git.lix.dev</li>
    <li>lixPackageSets.git.lix.devdoc</li>
    <li>lixPackageSets.git.lix.doc</li>
    <li>lixPackageSets.git.lix.man</li>
    <li>lixPackageSets.git.nix-direnv</li>
    <li>lixPackageSets.git.nix-eval-jobs</li>
    <li>lixPackageSets.git.nix-fast-build</li>
    <li>lixPackageSets.git.nix-fast-build.dist</li>
    <li>lixPackageSets.git.nix-init</li>
    <li>lixPackageSets.git.nix-update</li>
    <li>lixPackageSets.git.nix-update.dist</li>
    <li>lixPackageSets.git.nixpkgs-review</li>
    <li>lixPackageSets.git.nixpkgs-review.dist</li>
    <li>lixPackageSets.latest.colmena (lixPackageSets.lix_2_94.colmena)</li>
    <li>lixPackageSets.latest.lix (lixPackageSets.lix_2_94.lix)</li>
    <li>lixPackageSets.latest.lix.debug (lixPackageSets.lix_2_94.lix.debug)</li>
    <li>lixPackageSets.latest.lix.dev (lixPackageSets.lix_2_94.lix.dev)</li>
    <li>lixPackageSets.latest.lix.devdoc (lixPackageSets.lix_2_94.lix.devdoc)</li>
    <li>lixPackageSets.latest.lix.doc (lixPackageSets.lix_2_94.lix.doc)</li>
    <li>lixPackageSets.latest.lix.man (lixPackageSets.lix_2_94.lix.man)</li>
    <li>lixPackageSets.latest.nix-direnv (lixPackageSets.lix_2_94.nix-direnv)</li>
    <li>lixPackageSets.latest.nix-eval-jobs (lixPackageSets.lix_2_94.nix-eval-jobs)</li>
    <li>lixPackageSets.latest.nix-fast-build (lixPackageSets.lix_2_94.nix-fast-build)</li>
    <li>lixPackageSets.latest.nix-fast-build.dist (lixPackageSets.lix_2_94.nix-fast-build.dist)</li>
    <li>lixPackageSets.latest.nix-init (lixPackageSets.lix_2_94.nix-init)</li>
    <li>lixPackageSets.latest.nix-update (lixPackageSets.lix_2_94.nix-update)</li>
    <li>lixPackageSets.latest.nix-update.dist (lixPackageSets.lix_2_94.nix-update.dist)</li>
    <li>lixPackageSets.latest.nixpkgs-review (lixPackageSets.lix_2_94.nixpkgs-review)</li>
    <li>lixPackageSets.latest.nixpkgs-review.dist (lixPackageSets.lix_2_94.nixpkgs-review.dist)</li>
    <li>lixPackageSets.stable.colmena (lixPackageSets.lix_2_93.colmena)</li>
    <li>lixPackageSets.stable.nix-direnv (lixPackageSets.lix_2_93.nix-direnv)</li>
    <li>lixPackageSets.stable.nix-eval-jobs (lixPackageSets.lix_2_93.nix-eval-jobs)</li>
    <li>lixPackageSets.stable.nix-fast-build (lixPackageSets.lix_2_93.nix-fast-build)</li>
    <li>lixPackageSets.stable.nix-fast-build.dist (lixPackageSets.lix_2_93.nix-fast-build.dist)</li>
    <li>lixPackageSets.stable.nix-init (lixPackageSets.lix_2_93.nix-init)</li>
    <li>lixPackageSets.stable.nix-update (lixPackageSets.lix_2_93.nix-update)</li>
    <li>lixPackageSets.stable.nix-update.dist (lixPackageSets.lix_2_93.nix-update.dist)</li>
    <li>lixPackageSets.stable.nixpkgs-review (lixPackageSets.lix_2_93.nixpkgs-review)</li>
    <li>lixPackageSets.stable.nixpkgs-review.dist (lixPackageSets.lix_2_93.nixpkgs-review.dist)</li>
    <li>lixStatic</li>
    <li>lixStatic.dev</li>
    <li>localtunnel</li>
    <li>lua51Packages.luarocks-nix</li>
    <li>luarocks-nix (luaPackages.luarocks-nix)</li>
    <li>lua53Packages.luarocks-nix</li>
    <li>lua54Packages.luarocks-nix</li>
    <li>luajitPackages.luarocks-nix</li>
    <li>luarocks-packages-updater</li>
    <li>luarocks-packages-updater.dist</li>
    <li>lux-cli</li>
    <li>mdbook</li>
    <li>mdwatch</li>
    <li>meshlab</li>
    <li>mlv-app</li>
    <li>nil</li>
    <li>nim_lk</li>
    <li>niv (niv.bin, niv.data)</li>
    <li>nix (nixVersions.nix_2_31, nixVersions.stable)</li>
    <li>nix-bundle</li>
    <li>nix-direnv</li>
    <li>nix-du</li>
    <li>nix-eval-jobs</li>
    <li>nix-eval-jobs.dev</li>
    <li>nix-fast-build</li>
    <li>nix-fast-build.dist</li>
    <li>nix-forecast</li>
    <li>nix-heuristic-gc</li>
    <li>nix-heuristic-gc.dist</li>
    <li>nix-index</li>
    <li>nix-init</li>
    <li>nix-inspect</li>
    <li>nix-pin</li>
    <li>nix-plugins</li>
    <li>nix-prefetch</li>
    <li>nix-prefetch-docker</li>
    <li>nix-prefetch-github (python313Packages.nix-prefetch-github)</li>
    <li>nix-prefetch-github.dist (python313Packages.nix-prefetch-github.dist)</li>
    <li>nix-prefetch-github.man (python313Packages.nix-prefetch-github.man)</li>
    <li>nix-required-mounts</li>
    <li>nix-required-mounts.dist</li>
    <li>nix-serve</li>
    <li>nix-serve-ng</li>
    <li>nix-template</li>
    <li>nix-unit</li>
    <li>nix-update</li>
    <li>nix-update.dist</li>
    <li>nix-visualize</li>
    <li>nix-visualize.dist</li>
    <li>nix-web</li>
    <li>nix.dev (nixVersions.nix_2_31.dev, nixVersions.stable.dev)</li>
    <li>nix.doc (nixVersions.nix_2_31.doc, nixVersions.stable.doc)</li>
    <li>nix.man (nixVersions.nix_2_31.man, nixVersions.stable.man)</li>
    <li>nixVersions.latest (nixVersions.nix_2_32)</li>
    <li>nixVersions.latest.dev (nixVersions.nix_2_32.dev)</li>
    <li>nixVersions.latest.doc (nixVersions.nix_2_32.doc)</li>
    <li>nixVersions.latest.man (nixVersions.nix_2_32.man)</li>
    <li>nixVersions.nix_2_28</li>
    <li>nixVersions.nix_2_28.debug</li>
    <li>nixVersions.nix_2_28.dev</li>
    <li>nixVersions.nix_2_28.doc</li>
    <li>nixVersions.nix_2_28.man</li>
    <li>nixVersions.nix_2_29</li>
    <li>nixVersions.nix_2_29.dev</li>
    <li>nixVersions.nix_2_29.doc</li>
    <li>nixVersions.nix_2_29.man</li>
    <li>nixVersions.nix_2_30</li>
    <li>nixVersions.nix_2_30.dev</li>
    <li>nixVersions.nix_2_30.doc</li>
    <li>nixVersions.nix_2_30.man</li>
    <li>nixci</li>
    <li>nixd</li>
    <li>nixos-anywhere</li>
    <li>nixos-generators</li>
    <li>nixos-option</li>
    <li>nixos-rebuild-ng</li>
    <li>nixos-rebuild-ng.dist</li>
    <li>nixos-shell</li>
    <li>nixpkgs-hammering</li>
    <li>nixpkgs-manual</li>
    <li>nixpkgs-review</li>
    <li>nixpkgs-review.dist</li>
    <li>nixpkgs-reviewFull</li>
    <li>nixpkgs-reviewFull.dist</li>
    <li>nixseparatedebuginfod2</li>
    <li>nixt</li>
    <li>nixt.dev</li>
    <li>nixtract</li>
    <li>node2nix (nodePackages.node2nix)</li>
    <li>nodePackages_latest.node2nix</li>
    <li>nps</li>
    <li>nuget-to-json</li>
    <li>nurl</li>
    <li>nvfetcher</li>
    <li>oboete</li>
    <li>omnix</li>
    <li>onagre</li>
    <li>pop-launcher</li>
    <li>python312Packages.nix-kernel</li>
    <li>python312Packages.nix-kernel.dist</li>
    <li>python312Packages.nix-prefetch-github</li>
    <li>python312Packages.nix-prefetch-github.dist</li>
    <li>python312Packages.nix-prefetch-github.man</li>
    <li>python312Packages.pymeshlab</li>
    <li>python313Packages.nix-kernel</li>
    <li>python313Packages.nix-kernel.dist</li>
    <li>python313Packages.nixpkgs-updaters-library</li>
    <li>python313Packages.nixpkgs-updaters-library.dist</li>
    <li>python313Packages.pymeshlab</li>
    <li>quick-webapps</li>
    <li>ragenix</li>
    <li>rofi-games</li>
    <li>sbomnix</li>
    <li>sbomnix.dist</li>
    <li>ssh-openpgp-auth</li>
    <li>sshd-openpgp-auth</li>
    <li>sus-compiler</li>
    <li>swiftpm2nix (swiftPackages.swiftpm2nix)</li>
    <li>tasks</li>
    <li>terranix</li>
    <li>tests.dhall.generateDhallDirectoryPackage</li>
    <li>tests.fetchgit.cached-prefetch-avoids-fetch</li>
    <li>tests.fetchgit.prefetch-git-no-add-path</li>
    <li>tests.haskell.cabalSdist.hercules-ci-cnix-store</li>
    <li>tests.haskell.cabalSdist.hercules-ci-cnix-store.doc</li>
    <li>tests.pkg-config.defaultPkgConfigPackages.nix-cmd</li>
    <li>tests.pkg-config.defaultPkgConfigPackages.nix-expr</li>
    <li>tests.pkg-config.defaultPkgConfigPackages.nix-main</li>
    <li>tests.pkg-config.defaultPkgConfigPackages.nix-store</li>
    <li>tests.pkgs-lib.formats</li>
    <li>tests.pkgs-lib.formats-tests</li>
    <li>tests.testers.lycheeLinkCheck.network</li>
    <li>tests.testers.nixosTest-example</li>
    <li>tests.testers.runNixOSTest-example (tests.testers.runNixOSTest-extendNixOS)</li>
    <li>tests.trivial-builders.references</li>
    <li>turnon</li>
    <li>update-nix-fetchgit</li>
    <li>update-python-libraries</li>
    <li>vclab-nexus</li>
    <li>vclab-nexus.bin</li>
    <li>vimPluginsUpdater</li>
    <li>vulnix</li>
    <li>vulnix.dist</li>
    <li>vulnix.doc</li>
    <li>vulnix.man</li>
    <li>wp4nix</li>
    <li>xdg-desktop-portal-cosmic</li>
    <li>xdg-desktop-portal-cosmic.debug</li>
    <li>xosd-xft</li>
    <li>zon2nix</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>engage</summary>
<pre>test tasks_dependency_cycle_self_loop_dot ... ok
test tasks_dependency_cycle_self_loop ... ok
test run_specific_task_with_deps ... ok
test try_nonexistent_task ... ok
test try_nonexistent_both ... ok
test try_nonexistent_group ... ok

test result: ok. 37 passed; 0 failed; 0 ignored; 0 measured; 1 filtered out; finished in 0.14s

Finished cargoCheckHook
Running phase: installPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;installPhase&quot; }
Executing cargoInstallHook
ERROR Invalid configuration file
	Caused by: TOML parse error at line 3, column 1
  |
3 | multilingual = false
  | ^^^^^^^^^^^^
unknown field `multilingual`, expected one of `title`, `authors`, `description`, `src`, `language`, `text-direction`
</pre>
</details>
<details>
<summary>helix</summary>
<pre>   |               |
   |               the lifetime is elided here
   |
   = help: the same lifetime is referred to in inconsistent ways, making the signature confusing
help: use `&#x27;_` for type paths
   |
16 |     fn format(&amp;self, data: &amp;Self::Data) -&gt; Row&lt;&#x27;_&gt;;
   |                                               ++++

warning: `helix-term` (lib) generated 5 warnings (run `cargo fix --lib -p helix-term` to apply 5 suggestions)
    Finished `release` profile [optimized] target(s) in 1m 28s
 INFO Book building has started
 INFO Running the html backend
 WARN unclosed HTML tag `&lt;n&gt;` found in `commands.md` while exiting TableCell
HTML tags must be closed before exiting a markdown element.
 WARN unclosed HTML tag `&lt;n&gt;` found in `commands.md` while exiting TableCell
HTML tags must be closed before exiting a markdown element.
ERROR Rendering failed
	Caused by: Error rendering &quot;index&quot; line 200, col 25: Helper not found previous
	Caused by: Helper not found previous</pre>
</details>
<details>
<summary>mdbook-cmdrun</summary>
<pre>ERROR Unable to run the preprocessor `cmdrun`
	Caused by: No such file or directory (os error 2)

note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

---- build_test_book_with_dynamic_src stdout ----

thread &#x27;build_test_book_with_dynamic_src&#x27; (3955) panicked at tests/book.rs:29:5:
assertion `left == right` failed
  left: 101
 right: 0


failures:
    build_test_book
    build_test_book_with_dynamic_src

test result: FAILED. 0 passed; 2 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.02s

error: test failed, to rerun pass `--test book`</pre>
</details>
<details>
<summary>tests.devShellTools.nixos</summary>
<pre>docker # [  324.181918] dockerd[893]: time=&quot;2025-12-17T01:20:37.365370004Z&quot; level=error msg=&quot;Error saving dying container to disk: invalid output path: stat /var/lib/docker/containers/bb17f317368a682f326eef9aa5557b29eb025a7e619869cb1c3de34ab04ca738: no such file or directory&quot;
docker # [  324.185733] dockerd[893]: time=&quot;2025-12-17T01:20:37.369197027Z&quot; level=error msg=&quot;Handler for POST /v1.52/containers/create returned error: apply layer error for \&quot;\&quot;: failed to prepare extraction snapshot \&quot;extract-356277228-QbuO sha256:775f6077b7121e2879ea7c0568a68e6b07568bcd75462c03c21cb2d213ff40a7\&quot;: failed to create prepare snapshot dir: failed to create temp dir: mkdir /var/lib/docker/containerd/daemon/io.containerd.snapshotter.v1.overlayfs/snapshots/new-1023917356: no space left on device&quot;
docker # docker: Error response from daemon: apply layer error for &quot;&quot;: failed to prepare extraction snapshot &quot;extract-356277228-QbuO sha256:775f6077b7121e2879ea7c0568a68e6b07568bcd75462c03c21cb2d213ff40a7&quot;: failed to create prepare snapshot dir: failed to create temp dir: mkdir /var/lib/docker/containerd/daemon/io.containerd.snapshotter.v1.overlayfs/snapshots/new-1023917356: no space left on device
docker # 
docker # Run &#x27;docker run --help&#x27; for more information
docker: output: 
!!! Test &quot;buildNixShellImage: can build derivations&quot; failed with error: &quot;command `docker run --rm -it nix-shell-build-derivation` failed (exit code 125)&quot;
!!! Traceback (most recent call last):
!!!   File &quot;&lt;string&gt;&quot;, line 58, in &lt;module&gt;
!!!     docker.succeed(
!!! 
!!! RequestedAssertionFailed: command `docker run --rm -it nix-shell-build-derivation` failed (exit code 125)
cleanup
kill machine (pid 9)
qemu-system-x86_64: terminating on signal 15 from pid 6 (/nix/store/3lll9y925zz9393sa59h653xik66srjb-python3-3.13.9/bin/python3.13)
vde_switch: EOF data port: Interrupted system call
kill vlan (pid 7)
vde_switch: EOF on stdin, cleaning up and exiting
vde_switch: Caught signal 15, cleaning up and exiting
(finished: cleanup, in 0.00 seconds)</pre>
</details>
</details>

roberth committed:
devenv: apply mdbook 0.5 support patch to devenv_nix

Apply the mdbook 0.5 support patch to devenv's bundled Nix 2.30.4.

The patch is from upstream Nix commit 5cbd7856, which maintains
compatibility with both mdbook 0.4.x and 0.5.x. The doc/manual/package.nix
file is excluded as it doesn't exist in cachix's Nix fork.

https://github.com/NixOS/nix/issues/14628

roberth committed:
nixVersions.git: 2.33pre20251107_479b6b73 -> 2.34pre20251217_b6add8dc

Update to latest Nix master which includes mdbook 0.5 support merged in
PR #14690.

https://github.com/NixOS/nix/pull/14690

philiptaron reviewed (commented):
Doing a build. Changes LGTM.

roberth committed:
helix: fix build with mdbook 0.5

Update helix documentation build for mdbook 0.5 compatibility.

Changes:
- Replace deprecated {{#previous}}/{{#next}} helpers with {{#if}} blocks
- Update link references from {{link}} to {{previous.link}}/{{next.link}}
- Escape HTML tags in command descriptions to prevent markdown parsing errors
- Fix book-html path in postInstall (../book-html instead of book-html)

The mdbook 0.5 release changed navigation helpers from block helpers to
simple objects, requiring template syntax updates in index.hbs.

roberth committed:
engage: fix build with mdbook 0.5

Update engage documentation build for mdbook 0.5 compatibility.

Changes:
- Remove deprecated 'multilingual' field from book.toml
- Update git-repository-icon from 'fa-git-square' to 'fab-square-git'

The 'multilingual' field was removed in mdbook 0.5 as it was never used.
The FontAwesome icon name changed from 'fa-git-square' (FA 4) to
'fab-square-git' (FA 5 brand icon) in the version bundled with mdbook 0.5.

philiptaron commented:
The upgrade tests are dumb, and I'm skipping them.

roberth committed:
mdbook-cmdrun: 0.6.0-unstable-2024-04-15 -> 0.7.3-unstable-2025-12-22

This supports mdbook 0.5.

RaitoBezarius reviewed (approved):
On the Lix changes:

- [x] tested stable and its docs
- [x] tested cross on stable and its docs

philiptaron commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 467009`
Commit: `bcbe96f8a64090e9d1d0ebf7be7e1f38602a9b3d`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 1 package marked as broken and skipped:</summary>
  <ul>
    <li>tests.pkg-config.defaultPkgConfigPackages.tests-combined</li>
  </ul>
</details>
<details>
  <summary>:fast_forward: 2 packages blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>crush</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 test built:</summary>
  <ul>
    <li>nixosTests.simple</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 388 packages built:</summary>
  <ul>
    <li>aba</li>
    <li>appvm</li>
    <li>attic-client</li>
    <li>bundix</li>
    <li>cabal2nix</li>
    <li>cached-nix-shell</li>
    <li>cachix (cachix.bin)</li>
    <li>catppuccin-cursors</li>
    <li>catppuccin-cursors.frappeBlue</li>
    <li>catppuccin-cursors.frappeDark</li>
    <li>catppuccin-cursors.frappeFlamingo</li>
    <li>catppuccin-cursors.frappeGreen</li>
    <li>catppuccin-cursors.frappeLavender</li>
    <li>catppuccin-cursors.frappeLight</li>
    <li>catppuccin-cursors.frappeMaroon</li>
    <li>catppuccin-cursors.frappeMauve</li>
    <li>catppuccin-cursors.frappePeach</li>
    <li>catppuccin-cursors.frappePink</li>
    <li>catppuccin-cursors.frappeRed</li>
    <li>catppuccin-cursors.frappeRosewater</li>
    <li>catppuccin-cursors.frappeSapphire</li>
    <li>catppuccin-cursors.frappeSky</li>
    <li>catppuccin-cursors.frappeTeal</li>
    <li>catppuccin-cursors.frappeYellow</li>
    <li>catppuccin-cursors.latteBlue</li>
    <li>catppuccin-cursors.latteDark</li>
    <li>catppuccin-cursors.latteFlamingo</li>
    <li>catppuccin-cursors.latteGreen</li>
    <li>catppuccin-cursors.latteLavender</li>
    <li>catppuccin-cursors.latteLight</li>
    <li>catppuccin-cursors.latteMaroon</li>
    <li>catppuccin-cursors.latteMauve</li>
    <li>catppuccin-cursors.lattePeach</li>
    <li>catppuccin-cursors.lattePink</li>
    <li>catppuccin-cursors.latteRed</li>
    <li>catppuccin-cursors.latteRosewater</li>
    <li>catppuccin-cursors.latteSapphire</li>
    <li>catppuccin-cursors.latteSky</li>
    <li>catppuccin-cursors.latteTeal</li>
    <li>catppuccin-cursors.latteYellow</li>
    <li>catppuccin-cursors.macchiatoBlue</li>
    <li>catppuccin-cursors.macchiatoDark</li>
    <li>catppuccin-cursors.macchiatoFlamingo</li>
    <li>catppuccin-cursors.macchiatoGreen</li>
    <li>catppuccin-cursors.macchiatoLavender</li>
    <li>catppuccin-cursors.macchiatoLight</li>
    <li>catppuccin-cursors.macchiatoMaroon</li>
    <li>catppuccin-cursors.macchiatoMauve</li>
    <li>catppuccin-cursors.macchiatoPeach</li>
    <li>catppuccin-cursors.macchiatoPink</li>
    <li>catppuccin-cursors.macchiatoRed</li>
    <li>catppuccin-cursors.macchiatoRosewater</li>
    <li>catppuccin-cursors.macchiatoSapphire</li>
    <li>catppuccin-cursors.macchiatoSky</li>
    <li>catppuccin-cursors.macchiatoTeal</li>
    <li>catppuccin-cursors.macchiatoYellow</li>
    <li>catppuccin-cursors.mochaBlue</li>
    <li>catppuccin-cursors.mochaDark</li>
    <li>catppuccin-cursors.mochaFlamingo</li>
    <li>catppuccin-cursors.mochaGreen</li>
    <li>catppuccin-cursors.mochaLavender</li>
    <li>catppuccin-cursors.mochaLight</li>
    <li>catppuccin-cursors.mochaMaroon</li>
    <li>catppuccin-cursors.mochaMauve</li>
    <li>catppuccin-cursors.mochaPeach</li>
    <li>catppuccin-cursors.mochaPink</li>
    <li>catppuccin-cursors.mochaRed</li>
    <li>catppuccin-cursors.mochaRosewater</li>
    <li>catppuccin-cursors.mochaSapphire</li>
    <li>catppuccin-cursors.mochaSky</li>
    <li>catppuccin-cursors.mochaTeal</li>
    <li>catppuccin-cursors.mochaYellow</li>
    <li>catppuccin-sddm</li>
    <li>celeste</li>
    <li>chirpstack-concentratord</li>
    <li>colmena</li>
    <li>comma</li>
    <li>common-updater-scripts</li>
    <li>corto</li>
    <li>cosmic-applets</li>
    <li>cosmic-applibrary</li>
    <li>cosmic-bg</li>
    <li>cosmic-comp</li>
    <li>cosmic-comp.debug</li>
    <li>cosmic-design-demo</li>
    <li>cosmic-edit</li>
    <li>cosmic-ext-applet-caffeine</li>
    <li>cosmic-ext-applet-external-monitor-brightness</li>
    <li>cosmic-ext-applet-minimon</li>
    <li>cosmic-ext-applet-privacy-indicator</li>
    <li>cosmic-ext-applet-sysinfo</li>
    <li>cosmic-ext-applet-weather</li>
    <li>cosmic-ext-calculator</li>
    <li>cosmic-ext-tweaks</li>
    <li>cosmic-files</li>
    <li>cosmic-greeter</li>
    <li>cosmic-icons</li>
    <li>cosmic-idle</li>
    <li>cosmic-initial-setup</li>
    <li>cosmic-launcher</li>
    <li>cosmic-notifications</li>
    <li>cosmic-osd</li>
    <li>cosmic-panel</li>
    <li>cosmic-player</li>
    <li>cosmic-randr</li>
    <li>cosmic-reader</li>
    <li>cosmic-screenshot</li>
    <li>cosmic-session</li>
    <li>cosmic-settings</li>
    <li>cosmic-store</li>
    <li>cosmic-term</li>
    <li>cosmic-workspaces-epoch</li>
    <li>cosmic-workspaces-epoch.debug</li>
    <li>crate2nix</li>
    <li>devenv</li>
    <li>dhall-nix</li>
    <li>dhall-nixpkgs</li>
    <li>disko</li>
    <li>dogdns</li>
    <li>dogdns.man</li>
    <li>dub-to-nix</li>
    <li>dydisnix</li>
    <li>engage</li>
    <li>engage.doc</li>
    <li>evil-helix</li>
    <li>examine</li>
    <li>find-billy</li>
    <li>forecast</li>
    <li>fusionInventory</li>
    <li>gcalcli</li>
    <li>gcalcli.dist</li>
    <li>gclient2nix</li>
    <li>glpi-agent</li>
    <li>haskellPackages.cachix</li>
    <li>haskellPackages.cachix.bin</li>
    <li>haskellPackages.cachix.doc</li>
    <li>haskellPackages.cli-nix</li>
    <li>haskellPackages.cli-nix.doc</li>
    <li>haskellPackages.dhall-nix</li>
    <li>haskellPackages.dhall-nix.doc</li>
    <li>haskellPackages.dhall-nixpkgs</li>
    <li>haskellPackages.dhall-nixpkgs.doc</li>
    <li>haskellPackages.hercules-ci-agent</li>
    <li>haskellPackages.hercules-ci-agent.doc</li>
    <li>haskellPackages.hercules-ci-cli</li>
    <li>haskellPackages.hercules-ci-cli.doc</li>
    <li>haskellPackages.hercules-ci-cnix-expr</li>
    <li>haskellPackages.hercules-ci-cnix-expr.doc</li>
    <li>haskellPackages.hercules-ci-cnix-store</li>
    <li>haskellPackages.hercules-ci-cnix-store.doc</li>
    <li>haskellPackages.hnix</li>
    <li>haskellPackages.hnix.doc</li>
    <li>haskellPackages.niv</li>
    <li>haskellPackages.niv.bin</li>
    <li>haskellPackages.niv.data</li>
    <li>haskellPackages.niv.doc</li>
    <li>haskellPackages.nix-paths</li>
    <li>haskellPackages.nix-paths.doc</li>
    <li>haskellPackages.nix-serve-ng</li>
    <li>haskellPackages.nix-serve-ng.doc</li>
    <li>haskellPackages.nix-thunk</li>
    <li>haskellPackages.nix-thunk.doc</li>
    <li>haskellPackages.nvfetcher</li>
    <li>haskellPackages.nvfetcher.doc</li>
    <li>haskellPackages.update-nix-fetchgit</li>
    <li>haskellPackages.update-nix-fetchgit.doc</li>
    <li>hci</li>
    <li>helix</li>
    <li>helix.doc</li>
    <li>hercules-ci-agent</li>
    <li>home-manager</li>
    <li>hydra</li>
    <li>hydra.doc</li>
    <li>just</li>
    <li>just.doc</li>
    <li>just.man</li>
    <li>kabeljau</li>
    <li>kcl</li>
    <li>libcosmicAppHook</li>
    <li>libnixxml</li>
    <li>lix (lixPackageSets.lix_2_93.lix, lixPackageSets.stable.lix)</li>
    <li>lix.debug (lixPackageSets.lix_2_93.lix.debug, lixPackageSets.stable.lix.debug)</li>
    <li>lix.dev (lixPackageSets.lix_2_93.lix.dev, lixPackageSets.stable.lix.dev)</li>
    <li>lix.devdoc (lixPackageSets.lix_2_93.lix.devdoc, lixPackageSets.stable.lix.devdoc)</li>
    <li>lix.doc (lixPackageSets.lix_2_93.lix.doc, lixPackageSets.stable.lix.doc)</li>
    <li>lix.man (lixPackageSets.lix_2_93.lix.man, lixPackageSets.stable.lix.man)</li>
    <li>lixPackageSets.git.colmena</li>
    <li>lixPackageSets.git.lix</li>
    <li>lixPackageSets.git.lix.debug</li>
    <li>lixPackageSets.git.lix.dev</li>
    <li>lixPackageSets.git.lix.devdoc</li>
    <li>lixPackageSets.git.lix.doc</li>
    <li>lixPackageSets.git.lix.man</li>
    <li>lixPackageSets.git.nix-direnv</li>
    <li>lixPackageSets.git.nix-eval-jobs</li>
    <li>lixPackageSets.git.nix-fast-build</li>
    <li>lixPackageSets.git.nix-fast-build.dist</li>
    <li>lixPackageSets.git.nix-init</li>
    <li>lixPackageSets.git.nix-update</li>
    <li>lixPackageSets.git.nix-update.dist</li>
    <li>lixPackageSets.git.nixpkgs-review</li>
    <li>lixPackageSets.git.nixpkgs-review.dist</li>
    <li>lixPackageSets.latest.colmena (lixPackageSets.lix_2_94.colmena)</li>
    <li>lixPackageSets.latest.lix (lixPackageSets.lix_2_94.lix)</li>
    <li>lixPackageSets.latest.lix.debug (lixPackageSets.lix_2_94.lix.debug)</li>
    <li>lixPackageSets.latest.lix.dev (lixPackageSets.lix_2_94.lix.dev)</li>
    <li>lixPackageSets.latest.lix.devdoc (lixPackageSets.lix_2_94.lix.devdoc)</li>
    <li>lixPackageSets.latest.lix.doc (lixPackageSets.lix_2_94.lix.doc)</li>
    <li>lixPackageSets.latest.lix.man (lixPackageSets.lix_2_94.lix.man)</li>
    <li>lixPackageSets.latest.nix-direnv (lixPackageSets.lix_2_94.nix-direnv)</li>
    <li>lixPackageSets.latest.nix-eval-jobs (lixPackageSets.lix_2_94.nix-eval-jobs)</li>
    <li>lixPackageSets.latest.nix-fast-build (lixPackageSets.lix_2_94.nix-fast-build)</li>
    <li>lixPackageSets.latest.nix-fast-build.dist (lixPackageSets.lix_2_94.nix-fast-build.dist)</li>
    <li>lixPackageSets.latest.nix-init (lixPackageSets.lix_2_94.nix-init)</li>
    <li>lixPackageSets.latest.nix-update (lixPackageSets.lix_2_94.nix-update)</li>
    <li>lixPackageSets.latest.nix-update.dist (lixPackageSets.lix_2_94.nix-update.dist)</li>
    <li>lixPackageSets.latest.nixpkgs-review (lixPackageSets.lix_2_94.nixpkgs-review)</li>
    <li>lixPackageSets.latest.nixpkgs-review.dist (lixPackageSets.lix_2_94.nixpkgs-review.dist)</li>
    <li>lixPackageSets.stable.colmena (lixPackageSets.lix_2_93.colmena)</li>
    <li>lixPackageSets.stable.nix-direnv (lixPackageSets.lix_2_93.nix-direnv)</li>
    <li>lixPackageSets.stable.nix-eval-jobs (lixPackageSets.lix_2_93.nix-eval-jobs)</li>
    <li>lixPackageSets.stable.nix-fast-build (lixPackageSets.lix_2_93.nix-fast-build)</li>
    <li>lixPackageSets.stable.nix-fast-build.dist (lixPackageSets.lix_2_93.nix-fast-build.dist)</li>
    <li>lixPackageSets.stable.nix-init (lixPackageSets.lix_2_93.nix-init)</li>
    <li>lixPackageSets.stable.nix-update (lixPackageSets.lix_2_93.nix-update)</li>
    <li>lixPackageSets.stable.nix-update.dist (lixPackageSets.lix_2_93.nix-update.dist)</li>
    <li>lixPackageSets.stable.nixpkgs-review (lixPackageSets.lix_2_93.nixpkgs-review)</li>
    <li>lixPackageSets.stable.nixpkgs-review.dist (lixPackageSets.lix_2_93.nixpkgs-review.dist)</li>
    <li>lixStatic</li>
    <li>lixStatic.dev</li>
    <li>localtunnel</li>
    <li>lua51Packages.luarocks-nix</li>
    <li>luarocks-nix (luaPackages.luarocks-nix)</li>
    <li>lua53Packages.luarocks-nix</li>
    <li>lua54Packages.luarocks-nix</li>
    <li>luajitPackages.luarocks-nix</li>
    <li>luarocks-packages-updater</li>
    <li>luarocks-packages-updater.dist</li>
    <li>lux-cli</li>
    <li>mdbook</li>
    <li>mdbook-cmdrun</li>
    <li>mdwatch</li>
    <li>meshlab</li>
    <li>mlv-app</li>
    <li>nil</li>
    <li>nim_lk</li>
    <li>niv (niv.bin, niv.data)</li>
    <li>nix (nixVersions.nix_2_31, nixVersions.stable)</li>
    <li>nix-bundle</li>
    <li>nix-direnv</li>
    <li>nix-du</li>
    <li>nix-eval-jobs</li>
    <li>nix-eval-jobs.dev</li>
    <li>nix-fast-build</li>
    <li>nix-fast-build.dist</li>
    <li>nix-forecast</li>
    <li>nix-heuristic-gc</li>
    <li>nix-heuristic-gc.dist</li>
    <li>nix-index</li>
    <li>nix-init</li>
    <li>nix-inspect</li>
    <li>nix-pin</li>
    <li>nix-plugins</li>
    <li>nix-prefetch</li>
    <li>nix-prefetch-docker</li>
    <li>nix-prefetch-github (python313Packages.nix-prefetch-github)</li>
    <li>nix-prefetch-github.dist (python313Packages.nix-prefetch-github.dist)</li>
    <li>nix-prefetch-github.man (python313Packages.nix-prefetch-github.man)</li>
    <li>nix-required-mounts</li>
    <li>nix-required-mounts.dist</li>
    <li>nix-serve</li>
    <li>nix-serve-ng</li>
    <li>nix-template</li>
    <li>nix-unit</li>
    <li>nix-update</li>
    <li>nix-update.dist</li>
    <li>nix-visualize</li>
    <li>nix-visualize.dist</li>
    <li>nix-web</li>
    <li>nix.dev (nixVersions.nix_2_31.dev, nixVersions.stable.dev)</li>
    <li>nix.doc (nixVersions.nix_2_31.doc, nixVersions.stable.doc)</li>
    <li>nix.man (nixVersions.nix_2_31.man, nixVersions.stable.man)</li>
    <li>nixStatic</li>
    <li>nixStatic.dev</li>
    <li>nixStatic.doc</li>
    <li>nixStatic.man</li>
    <li>nixVersions.git</li>
    <li>nixVersions.git.dev</li>
    <li>nixVersions.git.doc</li>
    <li>nixVersions.git.man</li>
    <li>nixVersions.latest (nixVersions.nix_2_32)</li>
    <li>nixVersions.latest.dev (nixVersions.nix_2_32.dev)</li>
    <li>nixVersions.latest.doc (nixVersions.nix_2_32.doc)</li>
    <li>nixVersions.latest.man (nixVersions.nix_2_32.man)</li>
    <li>nixVersions.nix_2_28</li>
    <li>nixVersions.nix_2_28.debug</li>
    <li>nixVersions.nix_2_28.dev</li>
    <li>nixVersions.nix_2_28.doc</li>
    <li>nixVersions.nix_2_28.man</li>
    <li>nixVersions.nix_2_29</li>
    <li>nixVersions.nix_2_29.dev</li>
    <li>nixVersions.nix_2_29.doc</li>
    <li>nixVersions.nix_2_29.man</li>
    <li>nixVersions.nix_2_30</li>
    <li>nixVersions.nix_2_30.dev</li>
    <li>nixVersions.nix_2_30.doc</li>
    <li>nixVersions.nix_2_30.man</li>
    <li>nixci</li>
    <li>nixd</li>
    <li>nixos-anywhere</li>
    <li>nixos-generators</li>
    <li>nixos-option</li>
    <li>nixos-rebuild-ng</li>
    <li>nixos-rebuild-ng.dist</li>
    <li>nixos-shell</li>
    <li>nixpkgs-hammering</li>
    <li>nixpkgs-manual</li>
    <li>nixpkgs-review</li>
    <li>nixpkgs-review.dist</li>
    <li>nixpkgs-reviewFull</li>
    <li>nixpkgs-reviewFull.dist</li>
    <li>nixseparatedebuginfod2</li>
    <li>nixt</li>
    <li>nixt.dev</li>
    <li>nixtract</li>
    <li>node2nix (nodePackages.node2nix)</li>
    <li>nodePackages_latest.node2nix</li>
    <li>nps</li>
    <li>nuget-to-json</li>
    <li>nurl</li>
    <li>nvfetcher</li>
    <li>oboete</li>
    <li>omnix</li>
    <li>onagre</li>
    <li>pop-launcher</li>
    <li>python312Packages.nix-kernel</li>
    <li>python312Packages.nix-kernel.dist</li>
    <li>python312Packages.nix-prefetch-github</li>
    <li>python312Packages.nix-prefetch-github.dist</li>
    <li>python312Packages.nix-prefetch-github.man</li>
    <li>python312Packages.pymeshlab</li>
    <li>python313Packages.nix-kernel</li>
    <li>python313Packages.nix-kernel.dist</li>
    <li>python313Packages.nixpkgs-updaters-library</li>
    <li>python313Packages.nixpkgs-updaters-library.dist</li>
    <li>python313Packages.pymeshlab</li>
    <li>quick-webapps</li>
    <li>ragenix</li>
    <li>rofi-games</li>
    <li>sbomnix</li>
    <li>sbomnix.dist</li>
    <li>ssh-openpgp-auth</li>
    <li>sshd-openpgp-auth</li>
    <li>sus-compiler</li>
    <li>swiftpm2nix (swiftPackages.swiftpm2nix)</li>
    <li>tasks</li>
    <li>terranix</li>
    <li>tests.devShellTools.nixos</li>
    <li>tests.dhall.generateDhallDirectoryPackage</li>
    <li>tests.fetchgit.cached-prefetch-avoids-fetch</li>
    <li>tests.fetchgit.prefetch-git-no-add-path</li>
    <li>tests.haskell.cabalSdist.hercules-ci-cnix-store</li>
    <li>tests.haskell.cabalSdist.hercules-ci-cnix-store.doc</li>
    <li>tests.pkg-config.defaultPkgConfigPackages.nix-cmd</li>
    <li>tests.pkg-config.defaultPkgConfigPackages.nix-expr</li>
    <li>tests.pkg-config.defaultPkgConfigPackages.nix-main</li>
    <li>tests.pkg-config.defaultPkgConfigPackages.nix-store</li>
    <li>tests.pkgs-lib.formats</li>
    <li>tests.pkgs-lib.formats-tests</li>
    <li>tests.testers.lycheeLinkCheck.network</li>
    <li>tests.testers.nixosTest-example</li>
    <li>tests.testers.runNixOSTest-example (tests.testers.runNixOSTest-extendNixOS)</li>
    <li>tests.trivial-builders.references</li>
    <li>turnon</li>
    <li>update-nix-fetchgit</li>
    <li>update-python-libraries</li>
    <li>vclab-nexus</li>
    <li>vclab-nexus.bin</li>
    <li>vimPluginsUpdater</li>
    <li>vulnix</li>
    <li>vulnix.dist</li>
    <li>vulnix.doc</li>
    <li>vulnix.man</li>
    <li>wp4nix</li>
    <li>xdg-desktop-portal-cosmic</li>
    <li>xdg-desktop-portal-cosmic.debug</li>
    <li>xosd-xft</li>
    <li>zon2nix</li>
  </ul>
</details>
